pydantic/main.py

```diff
@@ -446,7 +453,8 @@ def copy(
             else:
                 v = {
                     **dict(
-                        self._iter(
+                        _iter(
```

Comment:
Does this need to be a method? I noticed some speedup from making it a non-method function instead (presumably since it removes the instance lookup).

Reply:
(The same goes for `_get_value`, to an even greater extent given the number of times it is called.)